.circleci/config.yml

```diff
@@ -229,30 +229,16 @@ workflows:
           requires:
             - create-service-builder
       - test-evergreen-canary:
-          name: test-hilltop-sms
-          url: hilltop-theatre-terraform
-          path: sms-ticket-function
+          name: test-canary-https-invocation-hello
+          url: canary-https-invocation
+          path: hello
```

Comment:
```suggestion
          path: src/hello-function
```
Maybe?

Reply:
"src/" is implied in the job, but I did miss the "-function" part!